src/systems/personal-space-bubble.js

```diff
@@ -68,13 +68,7 @@ AFRAME.registerSystem("personal-space-bubble", {
   tick() {
     if (!this.data.enabled) return;
 
-    // Update matrix positions once for each space bubble and space invader
```

Comment:
Hmm, so.. I do think this ends up being superfulous in our current app, because the character controller happens to do this already AND the rigs are actually already in world space. We could probably accidentally introduce a bug down the road though if we repaterented rigs and/or updated their position/rotation without updating the matrix. Since it sounded like this was a big performacne bottleneck its probably worth it, but maybe we should drop a comment here for the one of us in the future that is investigating a bug in space bubble positioning :laughing: